R/probability_epidemic.R

```diff
@@ -3,47 +3,63 @@
 #'
 #' @description Calculates the probability a branching process will cause an
 #' epidemic (i.e. probability will fail to go extinct) based on R, k and
-#' initial cases
+#' initial cases.
 #'
 #' @param R A `number` specifying the R parameter (i.e. average secondary cases
-#' per infectious individual)
+#' per infectious individual).
 #' @param k A `number` specifying the  k parameter (i.e. overdispersion in
-#' offspring distribution from fitted negative binomial)
-#' @param num_init_infect A `count` specifying the number of initial infections
-#' @param ... [`dots`] not used, extra arguments supplied will cause a warning.
-#' @param epidist An `<epidist>` object. An S3 class for working with
-#' epidemiological parameters/distributions, see [`epiparameter::epidist()`]
+#' offspring distribution from fitted negative binomial).
+#' @param num_init_infect A `count` specifying the number of initial infections.
```

Comment:
Since "count" is not really a type in R. Would it be better to say "integer" OR "number"?

```suggestion
#' @param num_init_infect A `number` specifying the number of initial infections.
```

Reply:
Resolved in 60691f42cb14b50da0b136243162e8c0a377648d.